prod/onChainData128.8-validator1.yaml

```diff
@@ -1,10 +1,10 @@
 ETH-USD:
   discrepancy: 0.5 # allowed discrepancy level for price signing [%]
-  precision: 8 # we can choose any precision for value, but it will be stored with 8 decimals always
```

Comment:
we can not have long comments?

Reply:
it is set to 80 characters by default, would you like that increase that limit?

Reply:
well, let's use default